ca/certificate-authority.go

```diff
@@ -169,6 +190,14 @@ func NewCertificateAuthorityImpl(cadb core.CertificateAuthorityDatabase, config
 		NotAfter:   issuer.NotAfter,
 	}
 
+	// TODO(rlb@ipv.sx): Add a stop channel to shut down the CA gracefully
+	ca.signingQueue = make(chan signingRequest, signingQueueSize)
+	go func(src chan signingRequest) {
+		for req := range ca.signingQueue {
+			ca.sign(req)
```

Comment:
What happens if the CA is kill -9'ed or crashes?

How do we know to resume signing those that were in the queue?